index.js

```diff
@@ -115,12 +114,51 @@ var isPercyEnabled = true;
 module.exports = {
   name: 'ember-percy',
 
+  _clientInfo: function() {
+    if(!this._clientInfoCache) {
+      var version = require('./package.json').version;
+      this._clientInfoCache = `${this.name}/${version}`;
+    }
+
+    return this._clientInfoCache;
+  },
+
+  _environmentInfo: function() {
+    if(!this._environmentInfoCache) {
+      this._environmentInfoCache = [
+        `ember/${this._emberSourceVersion()}`,
+        `ember-cli/${this._emberCliVersion()}`
+      ].filter(function(el) {
+        return !el.match(/\/$/);
+      }).join('; ');
+    }
+
+    return this._environmentInfoCache;
+  },
+
+  _emberSourceVersion: function() {
+    try {
+      return require('ember-source/package.json').version;
+    } catch (e) {
+      return '';
```

Comment:
@fotinakis I think that is a good idea, because you're right, if this client is being called it's definitely ember so it doesn't seem wrong to include that even if we didn't manage to extract the version number.